backend/src/main/java/de/bund/digitalservice/ris/norms/adapter/output/database/repository/NormManifestationRepository.java

```diff
@@ -96,4 +99,10 @@ void updatePublishStateByManifestationEli(
     @Param("manifestationEli") String manifestationEli,
     @Param("publishState") NormPublishState publishState
   );
+
+  @NativeQuery(
+    value = "select DISTINCT ON (eli_norm_work) * from norm_manifestation ORDER BY eli_norm_work, eli_norm_manifestation DESC",
+    countQuery = "select COUNT(DISTINCT eli_norm_work) from norm_manifestation"
+  )
+  Page<NormManifestationDto> findDistinctOnWorkEliByOrderByWorkEliAsc(Pageable pageable);
```

Comment:
Didn't we think about also extracting the title on db level so that we can just do here a simple give me the work elis and the title of their latest expression/manifestation? I bet this query will take a lot if we load the XML of all of them (up to 100)

Reply:
Yes this implementation is quite inefficient, but that was on purpose to first get something out before doing any kind of performance work. Thats also why I immediately created https://digitalservicebund.atlassian.net/browse/RISDEV-8447 after I tested that endpoint and saw that a request takes around 6s.